libcodechecker/analyze/analysis_manager.py

```diff
@@ -237,13 +237,30 @@ def check(check_data):
             # sequence, whereas clang-tidy does not. We rewrite the file
             # names to contain no escape sequences for every analyzer.
             result_file = rh.analyzer_result_file.replace(r'\ ', ' ')
+            result_base = os.path.basename(result_file)
+            capture_analysis_output = False
+            if hasattr(args, 'capture_analysis_output'):
```

Comment:
It is not defined in case analysis is executed in quickcheck mode.  I needed to introduce this check because of that. In quickheck mode, this parameter is non-sense.